README.md

```diff
@@ -0,0 +1,19 @@
+# Tech Guide for Arlington, TX PTAs
+The purpose of this guide is to distribute peer-reviewed best practices regarding the use of technology in conducting PTA business within our council.
```

Comment:
Spell out acronyms on first use within parentheses i.e. PTA where as TX should be fine
 - Possibly a subsection with a list of acronyms used per document.